src/commands/new.ts

```diff
@@ -503,116 +544,155 @@ export default {
-    p()
-    heading("Now get cooking! 🍽")
-    igniteHeading()
-    // #endregion
-
-    // #region Print CLI command
-    const flags: Required<Options> = {
-      b: bname,
-      boilerplate: bname,
-      bundle: bundleIdentifier,
-      coloLoco,
-      debug,
-      git,
-      installDeps,
-      overwrite,
-      expo,
-      packager: packagerName,
-      targetPath,
-      useCache,
-      y: yname,
-      yes: yname,
+    hr()
+    p2()
+    p2("Need additional help?")
+    p2()
+    p2(`Join our Slack community at ${link("http://community.infinite.red.")}`)
+    p2()
+
+    hr()
+    p2()
+    p2("Now get cooking! 🍽")
+    command(`cd ${projectName}`)
+    if (!installDeps) command(packager.installCmd({ packagerName }))
+    command(`${packagerName} start`)
+
+    const isMac = process.platform === "darwin"
+    if (isMac) {
+      command(`${packager.runCmd("ios", packagerOptions)}`)
+    } else {
+      command(`${packager.runCmd("android", packagerOptions)}`)
     }
 
-    type Flag = keyof typeof flags
-    type FlagEntry = [key: Flag, value: Options[Flag]]
-
-    const privateFlags: Flag[] = [
-      "b",
-      "boilerplate",
-      "coloLoco",
-      "debug",
-      "expo",
-      "useCache",
-      "y",
-      "yes",
-    ]
-
-    const stringFlag = ([key, value]: FlagEntry) => `--${kebabCase(key)}=${value}`
-    const booleanFlag = ([key, value]: FlagEntry) =>
-      value ? `--${kebabCase(key)}` : `--${kebabCase(key)}=${value}`
-
-    const cliCommand = `npx ignite-cli new ${projectName} ${(Object.entries(flags) as FlagEntry[])
-      .filter(([key]) => privateFlags.includes(key) === false)
-      .filter(([, value]) => value !== undefined)
-      .map(([key, value]) =>
-        typeof value === "boolean" ? booleanFlag([key, value]) : stringFlag([key, value]),
-      )
-      .join(" ")}`
+    p2()
+    p2("With Expo:")
+    command(`cd ${projectName}`)
+    if (!installDeps) command(packager.installCmd({ packagerName }))
+    command(`${packager.runCmd("expo:start", packagerOptions)}`)
+    p2()
+    p2()
+    // #endregion
 
-    p(`In the future, if you'd like to skip the questions, you can run Ignite with these options:`)
-    command(`  ${cliCommand}`)
-    p()
+    // #region Infinite Red Plug
+    // #endregion
 
     // this is a hack to prevent the process from hanging
     // if there are any tasks left in the event loop
     // like I/O operations to process.stdout and process.stderr
     // see https://github.com/infinitered/ignite/issues/2084
     process.exit(0)
-    // #endregion
   },
 }
+
+function buildCliCommand(args: {
+  flags: Required<Options>
+  toolbox: GluegunToolbox
+  projectName: string
+}): string {
+  const { flags, toolbox, projectName } = args
+  const { strings } = toolbox
+  const { kebabCase } = strings
+
+  type Flag = keyof typeof flags
+  type FlagEntry = [key: Flag, value: Options[Flag]]
+
+  const privateFlags: Flag[] = [
+    "b",
+    "boilerplate",
+    "coloLoco",
```

Comment:
Not now, but before our full launch, let's dump any references to coloLoco in the CLI.

Reply:
This was pretty straightforward, so I threw it in here:

[feat(new): remove references to colo-loco](https://github.com/infinitered/ignite/pull/2119/commits/d294be6998c09167492e37ae7b9ef03408a4756b)